ycms/cms/templates/user_settings.html

```diff
@@ -0,0 +1,34 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% load render_bundle from webpack_loader %}
+{% block title %}
+    {% translate "Settings" %}
+{% endblock title %}
+{% block content %}
+    <div class="flex justify-evenly items-center">
+        <div class="w-1/3 bg-white shadow p-8 dark:bg-gray-800">

```

Comment:
```suggestion
        <div class="w-1/3 rounded-lg border border-gray-300 bg-white p-4 shadow-md flex flex-col h-full dark:bg-gray-800 dark:border-gray-700">
```

I think this is copied from the user creation form view, right? We've since pivoted to using these kind of boxes everywhere. Could you also change this for the other form, and the user creation and login forms? 😇 😅